pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -3651,6 +3652,83 @@ func TestImportControlJob(t *testing.T) {
 	})
 }
 
+// TestImportCancelAfterCompletion ensures that a canceled IMPORT job cleans up
+// even after the tables have been published i.e the job has been "completed".
+func TestImportCancelAfterCompletion(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	skip.UnderRace(t, "node liveness error")
```

Comment:
this test is bound to flake with `node liveness error`. Is there a way to prevent this?